webhook/defaults.go

```diff
@@ -0,0 +1,63 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"go.uber.org/zap"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type DefaultMutator struct {
+	Client  client.Client
+	Log     *zap.SugaredLogger
+	decoder *admission.Decoder
+}
+
+func (m *DefaultMutator) InjectDecoder(d *admission.Decoder) error {
+	m.decoder = d
+
+	return nil
+}
+
+func (m *DefaultMutator) Handle(ctx context.Context, req admission.Request) admission.Response {
```

Comment:
While this only mutates the duration for now, this is a good way to handle all defaults until we move to v1 CRD